### ISSUE TITLE
Requesting Dataset Access

### ISSUE THREAD

**Issue by rahulbarnwal** [xiongzihua/FashionAI_Keypoint#1]
@xiongzihua FashionAI-KeyPintsDetectionofApparel got over on 5th of july. I came across this competition last week only. Though i already signed up, i am unable to access their dataset probably because the competition is already over. Could you share the dataset with me if you still have it ?

**Comment by abeardear** [comment]
I will check if i can share the dataset. If it is allowed, I can only upload it to BaiduYun.

**Comment by rahulbarnwal** [comment]
Yeah, sounds good. I'll be using the data for academic research, just in
case if it matters.
